### ISSUE TITLE
Tiny-Imagenet results

### ISSUE THREAD

**Issue by mboudiaf** [PaulAlbert31/LabelNoiseCorrection#4]
Hi,
Thank you very much for your work and for sharing the code. I was able to reproduce your results with my own code on CIFAR10 and CIFAR100. However, I wasn't able to reproduce them on Tiny-Imagenet with the same setting. I noticed overfitting was occuring soon during training, despite using mixup. This may come from me not using the right data augmentations (using random horizontal flip and normalization). Is there any additional augmentation or regularization you used in the paper ?
Thank you again.

Malik

**Comment by EricArazo** [comment]
Hello Malik,
As you said we used random horizontal flips and normalization, but we used random crops as well: for CIFAR10/100 we padded the images with 4 pixels per side and for TinyImageNet (since the images are larger) we padded them with 8 pixels. 
Let me know if you have further questions.
Best,
Eric
